core/main.go

```diff
@@ -72,59 +100,70 @@ func main() {
 				RootCAs:            caCertPool,
 				InsecureSkipVerify: *insecureSkipVerify,
 			},
+			TLSHandshakeTimeout: 5 * time.Second,
```

Comment:
it would also be nice if this were configurable from the configuration.toml file